### PR TITLE
feat: Add average timer counter

### PR DESCRIPTION
**Pull request by bxcodec**
### Proposed changes: 
- Add timer counter for every request
- Add make file
  - Now to do test we can do: `make test`
  - Build in current directory : `make gubrak`
- Adding linter
  - ```
    make lint-prepare
    make lint
    ```

### CODE REVIEW

**Comment by bxcodec** [comment]
Example result in `std.Out`

```bash
Status  200
Status  200
Status  200
Status  200
Status  200
Status  200
Status  200
Status  200
Status  200
Status  200
========================
Total all time:  137.945079ms
Average duration/request 124.060355ms
Average time to DNSLookup/request 2.847366ms
Average time to open a connection/request 24.917737ms
Average time to build a request/request 404.3µs
Average time to get response/request 70.444µs
Average time delay between request made and response got/request 98.387209ms
Total request succeed : 10 of : 10
```

**Comment by wuriyanto48** [comment]
wooh,, that output 👍 ,,
@bxcodec  can we have a fancy output :D, but without external dependency,,
with full colour maybe :D.....

aaaa hahhaha, maybe next time,

File: cmd/gubrak/main.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"runtime"
 
-	"github.com/Bhinneka/gubrak"
+	"github.com/bxcodec/gubrak"
```
Comment (wuriyanto48) [inline cmd/gubrak/main.go]: @bxcodec , can you fix this package name :D?

Comment (bxcodec) [reply cmd/gubrak/main.go]: sure

**Pull request merged**